src/lib/viewers/BaseViewer.js

```diff
@@ -1176,6 +1178,12 @@ class BaseViewer extends EventEmitter {
         }
     }
 
+    handleAnnotationStagedEvent({ staged }) {
+        if (this.annotationControls && !staged) {
+            this.annotationControls.focus();
```

Comment:
What is the desired user experience here?

Reply:
> What is the desired user experience here?

I was listening `keydown` event on `regionButton`, so I have to focus that button after `cancel` etc. However, listening at the document level solves this problem. I'm going to remove all the focus code.